tests/testsuite/profiles.rs

```diff
@@ -323,6 +323,33 @@ fn profile_in_virtual_manifest_works() {
         .run();
 }
 
+#[cargo_test]
+fn profile_lto_string_bool_dev() {
+    let p = project()
+        .file(
+            "Cargo.toml",
+            r#"
+                [package]
+                name = "foo"
+                version = "0.0.1"
+
+                [profile.dev]
+                lto = "true"
+            "#,
+        )
+        .file("src/lib.rs", "")
+        .build();
+
+    p.cargo("build")
+        .with_status(101)
+        .with_stderr_contains(
```

Comment:
Thanks for the update. For a more robust test, we can assert the whole stderr.

```suggestion
        .with_stderr(
```